templates/tools/dockerfile/test/php7_jessie_x64/Dockerfile.template

```diff
@@ -17,6 +17,7 @@
   FROM debian:jessie
   
   <%include file="../../php7_deps.include"/>
+  <%include file="../../php_fpm_nginx.include"/>
```

Comment:
you also need to run tools/buildgen/generate_project.sh for this to take effect.